.devcontainer/Dockerfile

```
@@ -0,0 +1,20 @@
+FROM mcr.microsoft.com/vscode/devcontainers/base:debian-10
```

Would it be an option to go for Ubuntu instead of Debian? Even though Debian is smaller and has its benefits, I believe the community would have a larger base of Ubuntu users.
```suggestion
FROM mcr.microsoft.com/vscode/devcontainers/base:0-focal
```

---

I don't have a strong opinion, but most of the examples I looked at in https://github.com/microsoft/vscode-dev-containers/ were Debian, including https://github.com/microsoft/vscode-dev-containers/tree/master/containers/deno/.devcontainer, so it seemed like the more conventional choice.